From: Niels Thykier <niels@thykier.net>
Date: Thu, 15 Mar 2012 20:03:39 +0100
Subject: pdebuild-script

---
 pdebuild/eclipse-pdebuild.sh |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/pdebuild/eclipse-pdebuild.sh b/pdebuild/eclipse-pdebuild.sh
index 572cd11..4a756e8 100755
--- a/pdebuild/eclipse-pdebuild.sh
+++ b/pdebuild/eclipse-pdebuild.sh
@@ -124,7 +124,7 @@ buildDir=$PWD/build
 SDK=$buildDir/SDK
 homeDir=$buildDir/home
 workspaceDir=$homeDir/workspace
-datadir=`rpm --eval "%{_libdir}"`
+datadir=/usr/lib
 pdeBuildDir=$datadir/eclipse/dropins/sdk/plugins/org.eclipse.pde.build_@PDEBUILDVERSION@
 
 featureId=
